fastai/data/external.py

```diff
@@ -10,14 +10,14 @@
 
 # Cell
 @lru_cache(maxsize=None)
-def fastai_cfg():
+def fastai_cfg() -> Config: # Config that contains default download paths for `'data', 'model', 'storage' and 'archive'`
```

Comment:
```suggestion
def fastai_cfg() -> Config: # Config that contains default download paths for `data`, `model`, `storage` and `archive`
```

Reply:
I thought we wanted the doc string to have the information in the Config docment, and the docment to be removed?

Reply:
@warner-benjamin `Config` is fastcore, which we're not docmenting for right now. This is okay for me at least for now to merge this, and if we need to iterate later I'm okay with that.